conf/test_epicore.config

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Additionally create an nf-test script in the tests folder. See e.g. test_ionannotator. You might only need to copy paste the test script, replace the profile name with test_epicore (every configuration is loaded from this config file) and run `nf-test test tests/epicore.nf.test --profile=+docker --verbose`. A snapshot file is created, commit everything and these tests are then run on GH CI

Reply:
* Make sure you have the newest nf-core package installed, there is automatically nf-test installed. You also can run `prettier -w .` and `nf-core pipelines lint` to lint your code